apps/api/views.py

```diff
@@ -0,0 +1,45 @@
+from django.http import HttpResponse
+from django.shortcuts import get_object_or_404
+from rest_framework import serializers
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+
+from apps.api.permissions import HasUserAPIKey
+from apps.experiments.models import Experiment, Participant, ParticipantData
+
+
+class ExperimentSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    experiment_id = serializers.UUIDField(source="public_id")
+
+
+@api_view(["GET"])
+@permission_classes([HasUserAPIKey])
```

Comment:
we should also check the django model permissions e.g. `experiments.view_experiment`